### ISSUE TITLE
Coordinates and Position Documentation

### ISSUE THREAD

**Issue by TomTomRixRix** [IMSY-DKFZ/simpa#163]
For new SIMPA developers it is very difficult to figure all the distances, locations and axes out. As I also have trouble getting it right quite often, I think it is a good idea to provide more documentation for these aspects. For this issue, I have the feeling that comments of what functions do or variables mean is not enough, but a general introduction text to the positioning with clear figures would help a lot. There we could also introduce assumptions about with direction is the x, y or z axis and common nomenclature.

What currently makes it even more difficult is that the `get_detector_element_positions_accounting_for_device_position_mm` and `get_detector_element_positions_accounting_for_field_of_view` methods in `DetectionGeometryBase` have the same description although they are doing different things, which is confusing.

**Comment by cbender98** [comment]
I totally agree with you! It is very confusing for the user to get familiar with the coordinate systems, which are used.


ad `get_detector_element_positions*`: As far as I know, the functions do the following. I would alsu suggest to check, if this is correct and maybe change the comments accordingly. (Bold text denotes change suggestions)

- `*_base_mm`: 
"""
Defines the abstract positions of the detection elements in an arbitrary coordinate system. 
Typically, the **detector position** is defined as the origin.
...
"""

- `*_accounting_for_device_position_mm`:
"""
...
This method returns the absolute positions of the detection elements relative to the **global volume, where the origin is defined by the volume origin** ~~where the device position is defined by the following tag~~
         ~~Tags.DIGITAL_DEVICE_POSITION~~ 
:returns: ...
 """

- `*_accounting_for_field_of_view`:
"""
...
This method returns the positions of the detection elements relative to **the field of view, where the origin is  the FOV origin (i.e at the corner of the FOV)**~~where the device position is defined by the following tag~~
~~Tags.DIGITAL_DEVICE_POSITION~~
:returns: ...
"""

**Comment by TomTomRixRix** [comment]
What might be most confusing is that the reconstruction uses x,y,z in a different manner than simpa in general. A consistent coordinate system nomenclature would help a lot.

**Comment by TomTomRixRix** [comment]
We could add the following information to the README at the end of the `Simulation examples` section:

In general, the axes of simulated photoacoustic images are implemented in the following way:
      
![grafik](https://user-images.githubusercontent.com/13121269/212362601-d54fa04e-c232-4fcc-a98a-bc06162ffbf7.png)

Note that z is the depth direction, while x and y form the plane parallel to the skin or PA device membrane.
